homeassistant/components/modbus/climate.py

```diff
@@ -42,13 +43,19 @@
 SUPPORT_FLAGS = SUPPORT_TARGET_TEMPERATURE
 HVAC_MODES = [HVAC_MODE_AUTO]
 
+REGISTER_TYPE_HOLDING = "holding"
+REGISTER_TYPE_INPUT = "input"
```

Comment:
Please add the default constants prefix `DEFAULT_`

Reply:
You mean like `DEFAULT_REGISTER_TYPE_HOLDING`? I just want to clarify. Thanks.

Reply:
Yes exactly.